### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by kashifzahid97** [richardadonnell/Upwork-Job-Scraper#68]
I am not sure whats the issue but the scraper extension is showing 0 Jobs

**Comment by richardadonnell** [comment]
Hey 👋

Thanks for reaching out - the extension requires some configuration, to search for the terms you like, see video here: https://youtu.be/FMH1QU0lz0Y?si=GpFHIIUNL0yl8EI5&t=313

Hope this helps! Please let me know if you still run into any issues! 😊

RAD

**Comment by kashifzahid97** [comment]
yes i did watch the video and did it according to that but still getting no jobs

**Comment by richardadonnell** [comment]
... I'm not sure if this is the issue or not, but, random question ... is your Upwork profile non-english?

**Comment by kashifzahid97** [comment]
No my profile is in english


11/24/2024, 9:31:55 PM: Manual scrape initiated
11/24/2024, 9:31:55 PM: 11/24/2024, 9:31:55 PM: Job check completed for https://www.upwork.com/nx/search/jobs/?payment_verified=1&q=%28instantly%20OR%20smartlead%20OR%20apollo%20OR%20zoominfo%29%20AND%20%22cold%20email%20%22
11/24/2024, 9:31:55 PM: 11/24/2024, 9:31:55 PM: Job check completed for https://www.upwork.com/nx/search/jobs/?payment_verified=1&q=%28instantly%20OR%20smartlead%20OR%20apollo%20OR%20zoominfo%29%20AND%20%22cold%20email%20%22
11/24/2024, 9:31:55 PM: 11/24/2024, 9:31:55 PM: No jobs scraped or unexpected result
11/24/2024, 9:31:55 PM: 11/24/2024, 9:31:55 PM: No jobs scraped or unexpected result
11/24/2024, 9:31:51 PM: 11/24/2024, 9:31:51 PM: Starting job check...
11/24/2024, 9:31:51 PM: 11/24/2024, 9:31:51 PM: Starting job check...
11/24/2024, 9:31:19 PM: 11/24/2024, 9:31:19 PM: Job check completed for https://www.upwork.com/nx/search/jobs/?payment_verified=1&q=%28instantly%20OR%20smartlead%20OR%20apollo%20OR%20zoominfo%29%20AND%20%22cold%20email%20%22
11/24/2024, 9:31:19 PM: 11/24/2024, 9:31:19 PM: Job check completed for https://www.upwork.com/nx/search/jobs/?payment_verified=1&q=%28instantly%20OR%20smartlead%20OR%20apollo%20OR%20zoominfo%29%20AND%20%22cold%20email%20%22
11/24/2024, 9:31:19 PM: Manual scrape initiated
11/24/2024, 9:31:19 PM: 11/24/2024, 9:31:19 PM: No jobs scraped or unexpected result
11/24/2024, 9:31:19 PM: 11/24/2024, 9:31:19 PM: No jobs scraped or unexpected result
11/24/2024, 9:31:16 PM: 11/24/2024, 9:31:16 PM: Starting job check...
11/24/2024, 9:31:16 PM: 11/24/2024, 9:31:16 PM: Starting job check...
11/24/2024, 9:31:01 PM: Feed source saved: custom-search

**Comment by richardadonnell** [comment]
Thank you for the feedback, and the log - that was a big help. I've identified and fixed the bug now.

I checked your link, and confirmed I was not able to get any jobs to scrape either.

The error occurred when trying to process job skills that were missing text content. The scraper was attempting to call `.trim()` on undefined text content, which caused the TypeError. (Error when attempting to scrape an empty array for the "skills" list.)

I've added defensive checks to ensure we only process text content when it actually exists. This fix has been implemented in the latest version, so updating to the newest release should resolve the error. I've uploaded the new version to the [Chrome Extension Web Store](https://chromewebstore.google.com/detail/upwork-job-scraper-+-webh/mojpfejnpifdgjjknalhghclnaifnjkg).

Checked your custom search URL link again after the fix, and verified it's scraping jobs now.

ℹ️ **_Please note_**: any updates to the extension need to be manually checked by an employee at Google, due to a permission required for the scraper. This process usually takes anywhere from 30 mins to 24 hours (typically less than 12 hours). New version with the fix is 👉 `1.49`. This should hopefully be approved and available shortly.

Thank you again for your feedback and the activity log!

Please let me know if you run into any other issues!

**Comment by richardadonnell** [comment]
That was quick - only 20 mins or so, and the update has been approved on the Chrome Extension Web Store -> v1.49, [link here](https://chromewebstore.google.com/detail/upwork-job-scraper-+-webh/mojpfejnpifdgjjknalhghclnaifnjkg).

Make sure you are running the updated version, and it should work!

Happy hunting!

**Comment by kashifzahid97** [comment]
Thanks its fixed and working nicely now

**Comment by kashifzahid97** [comment]
is there a way to scrap the job person's details like location and also scrap reviews from previous jobs?
